src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj

```diff
@@ -174,6 +174,12 @@
     <None Update="SessionRecords\Test.Azure.Management.Logic.WorkflowsScenarioTests\ValidateWorkflow.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\Test.Azure.Management.Logic.WorkflowsScenarioTests\ListSwagger.json">
```

Comment:
I also do not see version bump for this change.
Update the version in .csproj file as well as AssemblyInfo.cs